Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -259,13 +273,67 @@ public class Klaviyo: NSObject  {
                 , "Property values must be of NSString, NSNumber, NSNull, NSDictionary, NSDate, or NSURL. Got: \(String(describing: properties[k as! NSCopying]))")
         }
     }
+}
+
+
+@_spi(KlaviyoPrivate)
```

Comment:
Just to make sure I follow:

`@_spi` is used to mark this class and its methods as internal and not intended for public usage by developers right?

Reply:
Yah I put it here because I'm not ready to expose the api yet but we would rip this particular one off when we are ready.

Reply:
Gotcha that makes total sense to me! Thanks!